test/transform-decorator-namespace.test.js

```diff
@@ -0,0 +1,110 @@
+const { test, snapshot } = require("node:test");
+const assert = require("node:assert");
+const path = require("node:path");
+const vm = require("node:vm");
+const { transformSync } = require("../dist/index.js");
+
+snapshot.setResolveSnapshotPath((testPath) => {
+	return path.join(
+		__dirname,
+		"snapshots",
+		`${path.basename(testPath)}.snapshot`,
+	);
+});
+
+test("should transform TypeScript class decorators with multiple decorators", (t) => {
+	const inputCode = `
+  @sealed
+  @log
+  class BugReport {
+    type = "report";
+    title: string;
+    constructor(t: string) {
+      this.title = t;
+    }
+  }
+
+  function sealed(constructor: Function) {
+    Object.seal(constructor);
+    Object.seal(constructor.prototype);
+  }
+
+  function log(constructor: Function) {
+    console.log("Creating instance of", constructor.name);
+  }
+
+  const report = new BugReport("Test");
+`;
+
+	const { code } = transformSync(inputCode, {
+		mode: "transform",
+		sourceMap: true,
+	});
+
+	t.assert.snapshot(code);
+
+	try {
+		const script = new vm.Script(code);
+		const context = vm.createContext({});
+		context.report = null;
+		script.runInContext(context);
+
+		assert.ok(context.report, "Report instance should exist");
+		assert.strictEqual(
+			context.report.type,
+			"report",
+			"Report type should be 'report'",
+		);
+		assert.strictEqual(
+			context.report.title,
+			"Test",
+			"Report title should be 'Test'",
+		);
+	} catch (err) {
+		console.error("Error executing script:", err);
```

Comment:
why catch error during test?

Reply:
Sorry, I was logging the test and forgot to remove it